### DISCUSSION TITLE
Swagger fails to load - missing role attribute

### DISCUSSION THREAD

**Discussion by faFrafa** [fastapi/fastapi#6213] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# In utils.py under fastapi.openapi I have to comment this line to prevent "Failed to load" in Swagger UI
#  operation_security.append({security_name: security_requirement.role})
```

### Description

My Swagger UI used to work well, but suddenly I get a "Failed to load API definition", with the error below:
```
 File "..../lib/python3.9/site-packages/fastapi/openapi/utils.py", line 84, in get_openapi_security_definitions
    operation_security.append({security_name: security_requirement.role})
AttributeError: 'SecurityRequirement' object has no attribute 'role'
```

If I comment the line in utils.py, the /docs works again.
`#  operation_security.append({security_name: security_requirement.role})
`

Everything in the API works, it's just the OpenAPI that fails, unless I comment the above line.

For the note, I use oauth2 scheme with scopes:

```
oauth2_scheme = OAuth2PasswordBearer(
    tokenUrl="/login",
    scopes={
        Role.GUEST["name"]: Role.GUEST["description"],
        Role.EDITOR["name"]: Role.EDITOR["description"],
        Role.MANAGER["name"]: Role.MANAGER["description"],
    },
)

```

### Operating System

macOS

### FastAPI Version

0.87.0

### Python Version

Python 3.9.6

**Comment by sehraramiz** [comment]
SecurityRequirement object has a scopes attribute
the actual line of code is [like this](https://github.com/tiangolo/fastapi/blob/63a5ffcf577abc6015d021f65857b3826d8db74e/fastapi/openapi/utils.py#L84)
```
operation_security.append({security_name: security_requirement.scopes})
```

did you change the fastapi/openapi/utils.py ? if so please provide a minimum runnable sample of your code.

**Comment by faFrafa** [comment]
Hi, no I didn't change fastapi/openapi/utils.py

Great, thanks for your reply, I have no idea why and what changed the utils.py, but that was the issue, the line somehow changed.

**Comment by faFrafa** [comment, marked as answer]
Wrong git manipulation has modified the fastapi/openapi/utils.py file and caused the error
